spec/helpers/storyblok_helper_spec.rb

```diff
@@ -82,17 +86,17 @@
         {
           "component" => "nested_page",
           "page_container" => [
-            { "component" => "button", "title" => "button1" },
-            { "component" => "button", "title" => "button2" },
+            { "component" => "button", "title" => "button1", "_editable" => "editable" },
+            { "component" => "button", "title" => "button2", "_editable" => "editable" },
           ],
-          "_editable" => "_editable",
+          "_editable" => editable,
         }.to_dot
       end
 
       it "render the right partial" do
         result = storyblok_helper.rubyblok_component_tag(blok: component_object)
 
-        expect(result.squish).to eq("<div> <a> button1 </a> <a> button2 </a> </div>")
+        expect(result.squish).to eq("<!--#storyblok\#{\"name\": \"editable_name\"}--><div> editable<a> button1 </a> editable<a> button2 </a> </div>")
```

Comment:
I think it is fine to test `editable` only one place (ie line 80), otherwise it shouldn't be tested as it makes the tests hard to read.